kitsune/kpi/models.py

```diff
@@ -16,6 +16,12 @@
 EXIT_SURVEY_NO_CODE = 'exit-survey:no'
 EXIT_SURVEY_DONT_KNOW_CODE = 'exit-survey:dont-know'
 
+CONTRIBUTOR_COHORT_CODE = 'contributor'
+KB_CONTRIBUTOR_COHORT_CODE = 'contributor:kb'
+KB_L10N_CONTRIBUTOR_COHORT_CODE = 'contributor:kb:l10n'
+SUPPORT_FORUM_HELPER_COHORT_CODE = 'contributor:support_forum:helper'
+AOA_CONTRIBUTOR_COHORT_CODE = 'contributor:aoa'
```

Comment:
I think I would make `contributor:kb` into `contributor:kb:english`. The way it is now makes me think that `contributor:kb:l10n` is a subset of `contributor:kb` (It isn't, right?). This is a minor issue though.